src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3681,17 +3684,18 @@ def webapp_up(cmd, name, resource_group_name=None, plan=None, location=None, sku
         _configure_default_logging(cmd, rg_name, name)
     else:  # for existing app if we might need to update the stack runtime settings
         if os_name.lower() == 'linux' and site_config.linux_fx_version != runtime_version:
-            logger.warning('Updating runtime version from %s to %s',
-                           site_config.linux_fx_version, runtime_version)
-            update_site_configs(cmd, rg_name, name, linux_fx_version=runtime_version)
-            logger.warning('Waiting for runtime version to propagate ...')
-            time.sleep(30)  # wait for kudu to get updated runtime before zipdeploy. Currently no way to poll for this
-        elif os_name.lower() == 'windows' and site_config.windows_fx_version != runtime_version:
```

Comment:
Removed this for windows since `windows_fx_version` is only used for windows container apps